setup.cfg

```
@@ -40,3 +40,6 @@ where = .
 include =
   fyntex
 exclude =
+
+[coverage:xml]
+output = test-reports/coverage/xml/coverage.xml
```

Move this to `.coveragerc.test.ini` and change it to:

```suggestion
[xml]
output = test_reports/coverage/xml/coverage.xml
```